bridge/opencensus/README.md

```diff
@@ -63,10 +67,56 @@ octrace.DefaultTracer = opencensus.NewTracer(tracer)
 
 Be sure to set the `Tracer` name to your instrumentation package name instead of `"bridge"`.
 
-### Incompatibilities
+#### Incompatibilities
 
 OpenCensus and OpenTelemetry APIs are not entirely compatible.  If the bridge finds any incompatibilities, it will log them.  Incompatibilities include:
 
 * Custom OpenCensus Samplers specified during StartSpan are ignored.
 * Links cannot be added to OpenCensus spans.
 * OpenTelemetry Debug or Deferred trace flags are dropped after an OpenCensus span is created.
+
+## Metrics
+
+### The problem: mixing libraries without mixing pipelines
+
+The problem for monitoring is simpler than the problem for tracing, since there
+are no context propagation issues to deal with. However, it still is difficult
+for users to migrate an entire applications' monitoring at once. It
+should be possible to send metrics generated by OpenCensus libraries to an 
+OpenTelemetry pipeline so that migrating a metric does not require maintaining
+separate export pipelines for OpenCensus and OpenTelemetry.
+
+### The Exporter "wrapper" solution
+
+The solution we use here is to allow wrapping an OpenTelemetry exporter such
+that it implements the OpenCensus exporter interfaces. This allows a single
+exporter to be used for metrics from *both* OpenCensus and OpenTelemetry.
+
+### User Journy
```

Comment:
```suggestion
### User Joureny
```

Reply:
This doesn't seem to have been resolved.

Reply:
applied